pkg/BUILD

```diff
@@ -37,7 +37,6 @@ py_library(
 py_binary(
     name = "build_tar",
     srcs = ["build_tar.py"],
-    python_version = "PY2",
```

Comment:
Please use python_version = PY3, for consistency with the rest.

Reply:
Okay!

Reply:
Should I change `make_rpm` to use `"PY3"` as well? Otherwise, it will be the only one in this file that uses `"PY2"`, except for `make_deb_py2`, which I figure should stay as is.

I notice several of the tests use `"PY2"` as well. Perhaps they should change also?
- [`archive_test`](https://github.com/bazelbuild/rules_pkg/blob/ce53425bc5449268ade670346bc39d8c52b1f822/pkg/tests/BUILD#L124)
- [`make_rpm_test`](https://github.com/bazelbuild/rules_pkg/blob/ce53425bc5449268ade670346bc39d8c52b1f822/pkg/tests/BUILD#L148)
- [`helpers_test`](https://github.com/bazelbuild/rules_pkg/blob/ce53425bc5449268ade670346bc39d8c52b1f822/pkg/tests/BUILD#L163)

Reply:
If you can update the tests, that would be great.

Reply:
Okay, done. I've updated everything to use `"PY3"` except the previously noted `make_deb_py2`.